lib/virtfs/xfs/file_class.rb

```diff
@@ -0,0 +1,224 @@
+module VirtFS::XFS
+  class FS
```

Comment:
@jerryk55 Yes, but I guess `fs` would work as well, since the class name is `FS`. Actually, the camcorderfs uses `fs` in the file names instead of `camcorderfs` or something like that. However, that was done to more easily distinguish the top and bottom layers of the recording boundary.